test/doctool/test-doctool-html.js

```diff
@@ -110,14 +113,16 @@ const testData = [
     file: fixtures.path('document_with_links.md'),
     html: '<h1>Usage and Example<span><a class="mark"' +
     'href="#foo_usage_and_example" id="foo_usage_and_example">#</a>' +
-    '</span></h1><h2>Usage<span><a class="mark" href="#foo_usage"' +
+    '</span></h1><section><h2>Usage<span><a class="mark" href="#foo_usage"' +
     'id="foo_usage">#</a></span></h2><p><code>node \\[options\\] index.js' +
     '</code></p><p>Please see the<a href="cli.html#cli-options">' +
-    'Command Line Options</a>document for more information.</p><h2>' +
+    'Command Line Options</a>document for more information.</p>' +
+    '</section><section><h2>' +
     'Example<span><a class="mark" href="#foo_example" id="foo_example">' +
     '#</a></span></h2><p>An example of a<a href="example.html">' +
     'webserver</a>written with Node.js which responds with<code>' +
-    '\'Hello, World!\'</code>:</p><h2>See also<span><a class="mark"' +
+    '\'Hello, World!\'</code>:</p></section><section>' +
+    '<h2>See also<span><a class="mark"' +
     'href="#foo_see_also" id="foo_see_also">#</a></span></h2><p>Check' +
     'out also<a href="https://nodejs.org/">this guide</a></p>'
```

Comment:
Should this have a closing `</section>` tag at the end of the string?

Reply:
I guess it doesn't matter, but if it's on one of the strings, it should be on both

Reply:
No strong opinions, I don't think it's necessary as long as the test passes – the test just checks if the expected string is included in the actual output.